src/routes/deleteMovieById.ts

```diff
@@ -0,0 +1,46 @@
+// external
+import { StatusCodes } from 'http-status-codes';
+
+// types & interfaces
+import { FastifyReply, RequestGenericInterface } from 'fastify';
+import { IStorage } from '../storage';
+
+
+interface requestGeneric extends RequestGenericInterface {
+    params: {
+        id: string
+    }
+}
+
+
+export default {
+    init: (httpClient: any, storage: IStorage) => {
+        if ( !httpClient || !storage ) {
```

Comment:
why not to import storage? i think this way would have been easier and more clear

Reply:
The idea was to offer to routes all needed instruments/dependencies from the caller code, to make routes maximum "dumb" and abstract from specific toolset.

But I think you're right and not every route needs storage access, for example. Fixed.